src/EditModel/EditDataEntryForm/processFormData.js

```diff
@@ -0,0 +1,102 @@
+import log from 'loglevel';
+
+// TODO: write tests for processFormData
+
+export const generateHtmlForId = (id, {
+    styleAttr,
+    disabledAttr,
+    insertGrey,
+    operands,
+    totals,
+    indicators,
+}) => {
+    const style = styleAttr ? ` style=${styleAttr}` : '';
+    const disabled = disabledAttr || insertGrey ? ' disabled="disabled"' : '';
+
+    if (id.includes('-')) {
+        const label = operands && operands[id];
+        const attr = `name="entryfield" title="${label}" value="[ ${label} ]"${style}${disabled}`.trim();
+        return `<input id="${id}" ${attr}/>`;
+    } else if (totals.hasOwnProperty(id)) {
+        const label = totals[id];
+        // TODO: `name="total"` is included twice - can we remove one of them or
```

Comment:
For some reason we are including the `name` attribute twice, resulting in `<input ... name="total" name="total">`. I did not fix this bug just in case it breaks something :shrug: